src-plugins/diffeomorphicDemons/diffeomorphicDemonsProcess.cpp

```diff
@@ -53,9 +52,9 @@ DiffeomorphicDemonsProcess::DiffeomorphicDemonsProcess() : itkProcessRegistratio
     d->updateFieldStandardDeviation = 0.0;
     d->displacementFieldStandardDeviation = 1.5;
     d->useHistogramMatching = false;
-    //set transform type for the exportation of the transformation to a file
-    this->setProperty("transformType","nonRigid");
-    setOutput(NULL);
+
+    // Gives the exported file type for medRegistrationSelectorToolBox
+    this->setProperty("transformType","notText");
```

Comment:
(This code hasn't changed since my previous comment but for some reason GitHub has removed it so I'll rewrite it):
Since this property does not store the type of the transformation being written anymore, but the type of the output file, it should be renamed to something like outputFileType.